### PR TITLE
add moved classes imports checker with auto correct context

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: com.liferay.blade.upgrade.liferay70/src/com/liferay/blade/upgrade/liferay70/apichanges/MovedClassesImport.java [hunk 1]
```diff
@@ -0,0 +1,1449 @@
...
+import com.liferay.blade.api.AutoMigrator;
+import com.liferay.blade.api.FileMigrator;
+import com.liferay.blade.upgrade.liferay70.ImportStatementMigrator;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+	property = {
+		"file.extensions=java",
+		"problem.title=Moved Classes Import Error",
```
Comment (gamerson) [inline com.liferay.blade.upgrade.liferay70/src/com/liferay/blade/upgrade/liferay70/apichanges/MovedClassesImport.java]: Is there not a breaking changes item we can link to?  could we add a section to breaking changes so this Migrator could link to some documentation?

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: com.liferay.blade.upgrade.liferay70/src/com/liferay/blade/upgrade/liferay70/apichanges/MovedClassesImport.java (hunk 1 above)
Comment (AndyWu2015) [inline com.liferay.blade.upgrade.liferay70/src/com/liferay/blade/upgrade/liferay70/apichanges/MovedClassesImport.java]: o yes we can add that to markdown doc.

**Pull request closed without merging**